projects/plugins/boost/app/assets/src/js/features/speed-score/speed-score.tsx

```diff
@@ -153,7 +159,10 @@ const SpeedScore = () => {
 				{ site.online && <PerformanceHistory /> }
 			</div>
 
-			<PopOut scoreChange={ showScoreChangePopOut } />
+			<PopOut
+				scoreChange={ showScoreChangePopOut }
+				scores={ { mobile: scores.current.mobile, desktop: scores.current.desktop } }
```

Comment:
Maybe this could just be the following instead?
```suggestion
				scores={ scores.current }
```

Reply:
I'll leave out the scores, as I had only added them for the Tracks event.